I!: added the initial version of DVC product documentation

gvyshnya opened the pull request:
The documentation pack comprises the elements below
- doc.rst - the source file with DVC product documentation in the format of RST text markup
- doc.html - the final HTML form of the documentation (ready to redistribute or publish on the site)
- cmd.bat - the command file that uses pandoc to convert doc.rst -> doc.html
- header.html - the HTML file with the code fragment to insert into HEADER tag of doc.html
- footer.html - the HTML file with the code fragment to insert into doc.html after the output produced by pandoc and before </body> tag
- top.html - the HTML file with the code fragment to insert into doc.html just after <body> tag, before the output generated by pandoc
- files in auxiliary folders (js, css, imaged) are used to create the appropriate look and feel of doc.html (they shall be redistributed along with doc.html)
-- Note: css/style.css is the place to edit major styles for DVC product documentation
-

dmpetrov commented on docs/cmd.bat:
```diff
@@ -0,0 +1,2 @@
+REM pandoc -s doc.rst -o doc_simple.html
+pandoc -s --toc -H header.html -B top.html -A footer.html doc.rst -o doc.html
```
Please add a new empty line at the end of the file.

dmpetrov commented on docs/doc.html:
```diff
@@ -0,0 +1,559 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```
`doc.html` file is not needed - it will be generated. Please remove.

gvyshnya commented:
I applied a bunch of changes in the code per your recent code review
- removed doc.html from the repo
- added the empty EL to cmd.bat
- decreased the error count detected by codeclimate from 37 to 6
-- removed prettifier.js and css (it was responsible for a big fraction of problems)
-- fixed some style definitions in style.css

**Note:**  remaining issues detected by codeclimate are not essential IMHO (like caring about support for IE6 and IE7 as of Q3 2017 :smile: etc.)
